// brmasker4/__ivy_ngcc__/dist/app.module.js
import { CommonModule } from '@angular/common';
import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
import { BrMaskerDirective } from './directives/brmasker4';
import * as ɵngcc0 from '@angular/core';
var BrMasker4Module = (function () {
    function BrMasker4Module() {
    }
BrMasker4Module.ɵfac = function BrMasker4Module_Factory(t) { return new (t || BrMasker4Module)(); };
BrMasker4Module.ɵmod = /*@__PURE__*/ ɵngcc0.ɵɵdefineNgModule({ type: BrMasker4Module });
BrMasker4Module.ɵinj = /*@__PURE__*/ ɵngcc0.ɵɵdefineInjector({ imports: [[
            CommonModule
        ]] });
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && ɵngcc0.ɵsetClassMetadata(BrMasker4Module, [{
        type: NgModule,
        args: [{
                declarations: [
                    BrMaskerDirective
                ],
                exports: [
                    BrMaskerDirective
                ],
                imports: [
                    CommonModule
                ],
                schemas: [
                    CUSTOM_ELEMENTS_SCHEMA
                ]
            }]
    }], function () { return []; }, null); })();
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(BrMasker4Module, { declarations: function () { return [BrMaskerDirective]; }, imports: function () { return [CommonModule]; }, exports: function () { return [BrMaskerDirective]; } }); })();
    return BrMasker4Module;
}());
export { BrMasker4Module };
/** @nocollapse */
BrMasker4Module.ctorParameters = function () { return []; };
